### PR TITLE
[OT185-20] Crear modelo y repositorio base de Testimonials

### PR DESCRIPTION
**Pull request by Daiana-Videla**
### Type of change
- [x ] New feature
- [ ] New Tests
- [ ] Bug fix
- [ ] Refactor

### Checklist

- [ ] Traceability between this change and Jira.
- [ ] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/domain/model/Testimonial.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "image", nullable = false)
+    private String image;
+
+    @Column(name = "content", nullable = false)
+    private String content;
+
+//prueba
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/model/Testimonial.java]: Por favor, no subamos este comentario

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,7 +2,7 @@
 spring.datasource.type=com.zaxxer.hikari.HikariDataSource
 spring.datasource.url=jdbc:mysql://localhost:3306/ongdb?createDatabaseIfNotExist=true&useUnicode=true&characterEncoding=UTF-8&serverTimezone=UTC
 spring.datasource.username=root
-#spring.datasource.password=password
+#spring.datasource.password=root
```
Comment (manerajona) [inline src/main/resources/application.properties]: Por favor, no subamos este cambio

**Review by manerajona — CHANGES_REQUESTED** [review]

File: src/main/java/com/alkemy/ong/domain/model/Testimonial.java (hunk 1 above)
Comment (Daiana-Videla) [reply src/main/java/com/alkemy/ong/domain/model/Testimonial.java]: borrado

File: src/main/resources/application.properties (hunk 2 above)
Comment (Daiana-Videla) [reply src/main/resources/application.properties]: modificado

Comment (manerajona) [reply src/main/resources/application.properties]: @Daiana-Videla tenes que revertir el cambio me sigue apareciendo modificado.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,7 +2,7 @@
 spring.datasource.type=com.zaxxer.hikari.HikariDataSource
 spring.datasource.url=jdbc:mysql://localhost:3306/ongdb?createDatabaseIfNotExist=true&useUnicode=true&characterEncoding=UTF-8&serverTimezone=UTC
 spring.datasource.username=root
-#spring.datasource.password=password
+spring.datasource.password=root
```
Comment (manerajona) [inline src/main/resources/application.properties]: Revertir este cambio.

**Review by manerajona — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 3 above)
Comment (Daiana-Videla) [reply src/main/resources/application.properties]: revertido

**Review by manerajona — APPROVED** [review]

**Pull request merged**